frontend/app/src/next/components/runs/run-dag/dag-run-visualizer.tsx

```diff
@@ -27,36 +29,40 @@ const edgeTypes = {
 
 interface WorkflowRunVisualizerProps {
   workflowRunId: string;
-  onTaskSelect?: (taskId: string) => void;
+  patchTask?: V1TaskSummary | null;
+  onTaskSelect?: (taskId: string, childWfrId?: string) => void;
```

Comment:
nit but also fine for now: in general I prefer to spell things out instead of using abbreviations if we can, for clarity

Reply:
👍 ez n-uff